draft-ietf-tls-dtls-connection-id.md

```diff
@@ -366,9 +369,9 @@ in changing a peer address if they are able to rewrite source addresses
 and if replayed packets are able to arrive before any original. 
 
 - There is a strategy for ensuring that the new peer address is able to 
-receive and process DTLS records. No such test is defined in this	specification. 
+receive and process DTLS records. No such test is defined in this specification.
 
-The above is necessary to protect against attacks that use datagrams with 
+The conditions above are necessary to protect against attacks that use datagrams with 
```

Comment:
I'm still confused about this. There are two values here:

- Don't change the path
- Don't process the packet

This text, however, seems to allow you to process it but not to respond, even to the original IP. What's the reason for that?

Reply:
I now see what creates the confusion. I have reworded the text and hope it is better now...

Reply:
LGTM